packages/runtime/runtime-utils/src/handles.ts

```diff
@@ -0,0 +1,23 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/**
+ * JSON serialized form of an IFluidHandle
+ * @internal
+ */
+export interface ISerializedHandle {
+	// Marker to indicate to JSON.parse that the object is a Fluid handle
```

Comment:
I don't think this is about JSON.parse. It's just how handle's are encoded and identified by the runtime.

Reply:
I think this comment was just copied.